src/atomate2/vasp/jobs/core.py

```diff
@@ -529,6 +531,40 @@ def make(
         return super().make.original(self, structure, prev_vasp_dir)
 
 
+@dataclass
+class MDMaker(BaseVaspMaker):
+    """
+    Maker to create VASP molecular dynamics jobs.
+
+    Parameters
+    ----------
+    name : str
+        The job name.
+    input_set_generator : .VaspInputSetGenerator
+        A generator used to make the input set.
+    write_input_set_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.write_vasp_input_set`.
+    copy_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.copy_vasp_outputs`.
+    run_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.run_vasp`.
+    task_document_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.TaskDocument.from_directory`.
+    stop_children_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.should_stop_children`.
+    write_additional_data : dict
+        Additional data to write to the current directory. Given as a dict of
+        {filename: data}. Note that if using FireWorks, dictionary keys cannot contain
+        the "." character which is typically used to denote file extensions. To avoid
+        this, use the ":" character, which will automatically be converted to ".". E.g.
+        ``{"my_file:txt": "contents of the file"}``.
+    """
+
+    # TODO expose basic setting of INCAR parameters to user
```

Comment:
Please don't put these controls in the job maker, instead put them in the input set generator to be consistent with the other input sets.

Also, I'd advise against exposing the basic incar settings as those are easily overriden using `user_incar_settings`. Instead you could add a several "presets", specified by a string, e.g., "nvt", "npt", "long" or whatever you think is useful